demo/index.html

```diff
@@ -136,10 +136,10 @@ <h4 class="centered">All pages animated the same way</h4>
               loading="{{loading}}"
               entry-animation="fade-in-animation"
               exit-animation="fade-out-animation">
-            <template is="iron-lazy-page" data-route="foo" path="element.html">
```

Comment:
Please revert this change. It is intended to test if nonexisting imports keep on working